src/Resources/translations/VerifyEmailBundle.de.xlf

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+    <file original="file.ext" source-language="en" datatype="plaintext">
+        <body>
+            <trans-unit id="1">
+                <source>%count% year|%count% years</source>
+                <target>%count% Jahr|%count% Jahren</target>
```

Comment:
In which context is this used? Depending on that `%count% Jahre` may be correct as well.

Reply:
applies below as well

Reply:
![CleanShot 2021-01-20 at 11 42 12](https://user-images.githubusercontent.com/995707/105163729-93148080-5b14-11eb-8dc9-531c9e213903.png)

In the confirmation email